tutorials/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -6,68 +6,53 @@ C# basics
+    improved. Please report issues with C# in Godot on the `engine GitHub page <https://github.com/godotengine/godot/issues>`_,
+    and any documentation issues on the `documentation GitHub page <https://github.com/godotengine/godot-docs/issues>`_.
 
 This page provides a brief introduction to C#, both what it is and
 how to use it in Godot. Afterwards, you may want to look at
 :ref:`how to use specific features <doc_c_sharp_features>`, read about the
-:ref:`differences between the C# and the GDScript API <doc_c_sharp_differences>`
+:ref:`differences between the C# and the GDScript API <doc_c_sharp_differences>`,
 and (re)visit the :ref:`Scripting section <doc_scripting>` of the
 step-by-step tutorial.
 
 C# is a high-level programming language developed by Microsoft. In Godot,
-it is implemented with the Mono 6.x .NET framework, including full support
-for C# 8.0. Mono is an open source implementation of Microsoft's .NET Framework
-based on the ECMA standards for C# and the Common Language Runtime.
-A good starting point for checking its capabilities is the
-`Compatibility <http://www.mono-project.com/docs/about-mono/compatibility/>`_
-page in the Mono documentation.
-
-.. note:: This is **not** a full-scale tutorial on the C# language as a whole.
-        If you aren't already familiar with its syntax or features,
-        see the
-        `Microsoft C# guide <https://docs.microsoft.com/en-us/dotnet/csharp/index>`_
-        or look for a suitable introduction elsewhere.
+it is implemented with .NET 6.0.
 
-.. _doc_c_sharp_setup:
-
-Setting up C# for Godot
------------------------
+.. note::
 
-Prerequisites
-~~~~~~~~~~~~~
+    This is **not** a full-scale tutorial on the C# language as a whole.
+    If you aren't already familiar with its syntax or features, see the
+    `Microsoft C# guide <https://docs.microsoft.com/en-us/dotnet/csharp/index>`_
+    or look for a suitable introduction elsewhere.
 
-Install the latest stable version of the
-`.NET SDK <https://dotnet.microsoft.com/download>`__, previously known as the
-.NET Core SDK.
+.. _doc_c_sharp_setup:
 
-As of Godot 3.2.3, installing Mono SDK is not a requirement anymore,
-except it is required if you are building the engine from source.
+Prerequisites
+-----------------------
 
-Godot bundles the parts of Mono needed to run already compiled games.
+Godot bundles the parts of .NET needed to run already compiled games.
 However, Godot does not bundle the tools required to build and compile
 games, such as MSBuild and the C# compiler. These are
-included in the .NET SDK, which needs to be installed separately.
+included in the .NET SDK, and need to be installed separately.
+
+In summary, you must have installed the .NET SDK **and** the .NET-enabled
+version of Godot.
 
-In summary, you must have installed the .NET SDK
-**and** the Mono-enabled version of Godot.
+Download and install the .NET 6.0 or .NET 7.0 SDK from the
```

Comment:
I'm not sure we need to mention the specific version, since now we roll-forward it should work with any version greater or equal to 6.0.